src/Compilers/CSharp/Test/Symbol/Symbols/UnsignedRightShiftTests.cs

```diff
@@ -0,0 +1,1873 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable disable
```

Comment:
Did I miss a langver test in here?

Reply:
> Did I miss a langver test in here?

I haven't touched the lang version part yet.